components/Sidemenu/Sidemenu.js

```diff
@@ -222,22 +225,31 @@ class Sidebar extends React.Component<Props, State> {
                 <MenuItem key={level} thinBorder>
                   <Trans>Level {level}</Trans>
                 </MenuItem>
-              </Link>
+              </RouteLink>
             ))}
-            <Link passHref route="new" params={{ lang: language.code }}>
+            <RouteLink passHref route="new" params={{ lang: language.code }}>
               <MenuItem thickBorder>
                 <Trans>New arrivals</Trans>
               </MenuItem>
-            </Link>
+            </RouteLink>
             <MenuItem href="https://home.digitallibrary.io/about/">
               <Trans>About Global Digital Library</Trans>
             </MenuItem>
             {config.TRANSLATION_PAGES && (
-              <Link passHref route="translations">
-                <MenuItem>
-                  <Trans>My translations</Trans>
-                </MenuItem>
-              </Link>
+              <React.Fragment>
+                <RouteLink passHref route="translations">
+                  <MenuItem>
+                    <Trans>My translations</Trans>
+                  </MenuItem>
+                </RouteLink>
+                {getAuthToken() != null && (
+                  <Link passHref href="/auth/sign-off">
+                    <MenuItem>
+                      <Trans>Sign out</Trans>
```

Comment:
Mener å huske å ha sett at innlogging heter `"Log in"`. Kanskje vi da skal også ha `Log out` i stedet for `Sign out`.